src/Search/Limit/MemoryLimit.php

```diff
@@ -24,4 +24,9 @@ public function __invoke() : bool
 
         return false;
     }
+
+    public function getReasons() : array
+    {
```

Comment:
Thinking about it, the original could be improved if it saved in $this->actualBytes the result of `memory_get_usage(true)`. That way we would be able to see if there are big jumps of memory like `Memory limit exceeded: 200000000 exceedes 100000000`